Fabric.Identity.API/scripts/Register-Identity-IdPSS.ps1

```diff
@@ -8,7 +8,8 @@ param(
         }
         return $true
     })] 
-    [string] $installConfigPath = "$PSScriptRoot\install.config"
+    [string] $installConfigPath = "$PSScriptRoot\install.config",
+	[string[]] $registerApps = ("Identity Service", "Identity Provider Search Service")
```

Comment:
Does it make sense to make these two into switches instead, similar to $quiet in install-identity

Reply:
sounds good